packages/generator-single-spa/src/root-config/templates/package.json

```diff
@@ -26,7 +26,6 @@
     "eslint-config-important-stuff": "^1.1.0",
     "eslint-config-prettier": "^7.0.0",
     "eslint-plugin-prettier": "^3.2.0",
-    "html-webpack-plugin": "^4.5.0",
```

Comment:
webpack-config-single-spa used to not depend on html-webpack-plugin, but it does now. So we no longer need this to be a dependency of the root config package.json directly.